apps/site/pages/docs/headless-cms-integration/2-setting-up-the-vtex-headless-cms.mdx

```diff
@@ -0,0 +1,173 @@
+| `content-types.json`     |  An array of JSON objects that describes the **Content Types** available for customization at the VTEX Headless CMS app.                                       |
+| `sections.json`          | An array of JSON objects that describes the content structure of the frontend **Section** components available for customization at the VTEX Headless CMS app. |
+|  `translation-keys.json` | an array of JSON objects that defines the translation keys of the Sections descriptions.                                                                       |
+
+5. Update the `sections.json` file with an empty array:
+
+```bash copy
+   echo "[]" > cms/faststore/sections.json
+```
+
+6. Update the `translation-keys.json` file with an empty object. :
+
+```bash copy
+   echo "{}" > cms/faststore/translation-keys.json
+```
+
+7. Open the `content-types.json` file in any code editor of your choice and add the following code:
+
+   ```json filename="cms/faststore/content-types.json" copy 
+    [
+       {
+        "id": "customPage",
+        "name": "Custom Page",
+        "configurationSchemaSets": [
+          {
+            "name": "Settings",
+            "configurations": [
+              {
+                "name": "seo",
+                "schema": {
+                  "title": "SEO",
+                  "description": "Search Engine Optimization options",
+                  "type": "object",
+                  "widget": {
+                    "ui:ObjectFieldTemplate": "GoogleSeoPreview"
+                  },
+                  "required": [
+                    "slug",
+                    "title",
+                    "description"
+                  ],
+                  "properties": {
+                    "slug": {
+                      "title": "Path",
+                      "type": "string",
+                      "default": "/"
+                    },
+                    "title": {
+                      "title": "Default page title",
+                      "description": "Display this title when no other title is available",
+                      "type": "string",
+                      "default": "FastStore Starter"
+                    },
+                    "description": {
+                      "title": "Meta tag description",
+                      "type": "string",
+                      "default": "A beautifully designed store"
+                    },
+                    "canonical": {
+                      "title": "Canonical url for the page",
+                      "type": "string"
+                    }
+                  }
+                }
+              }
+            ]
+          }
+        ]
+      }
+    ]
+
+   ```
+
+<Callout type="info" emoji="ℹ️">
+  Don't worry about the structure of this file for now, as we'll learn more about it later in this tutorial. 
+  However, notice that we have defined the **Custom Page** content type.
+</Callout>
+
+
+
+8. Save your changes in the `content-types.json` file.
```

Comment:
After step 8, should we run `yarn dev` to watch what the project already has in the core and the changes we are applying?